src/rp2_common/hardware_clocks/scripts/vcocalc.py

```diff
@@ -2,23 +2,32 @@
 
 import argparse
 
+# Fixed hardware parameters
+fbdiv_range = range(16, 320 + 1)
+postdiv_range = range(1, 7 + 1)
+ref_min = 5
+refdiv_min = 1
+refdiv_max = 63
+
+def validRefdiv(string):
+    if ((int(string) < refdiv_min) or (int(string) > refdiv_max)):
```

Comment:
Only a minor nit, but I think personally I'd do the `int(string)` once, rather than three separate times?